### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Database Properties Preview_Files_Keyboard: Unable to change column size with keyboard for "database files" table.

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/azuredatastudio#25132]
**"Please do not close this bug. This bug should only be closed by Trusted Tester after verification"**
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
## Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev1; #DesktopApp; #WCAG2.1.1; #FTP; #SH_Azure Data Studio_Nov2023_Win32_Dec2023; #SQL Azure Data Studio; #Win32; #Keyboard; #NonBenchmark;
## Environment Details:
Application Name: Azure Data Studio
Operating System: Win11
Azure data studio 1.47.0
## Repro Steps:
1. Install Azure data studio application and open it
3. Make sure to enable "Preview" feature in settings.
4. Tab till "Database" and press enter to expand the tree.
5. Tab till any database and pressF10 to open context menu.
6. Tab till "Properties Preview" and press enter.
7. Tab till "Files" tab and press enter.
8. Tab till "Database files" table and try to change column size.
9.  You will observe that unable to change column size with keyboard.
## Actual Result:
Unable to change column size with keyboard for "database files" table.
Note: Issue seen throughout the application for similar type control. 
## Expected Result:
User should be able to change column size with keyboard keys.
## User Impact:
Keyboard only user will unable change column size and see complete data. 
### Attachment:

https://github.com/microsoft/azuredatastudio/assets/124254981/567b9e18-e7a5-4180-b497-0f8bd4b3c047

**Comment by corivera** [comment]
@kapilvaishna resizing columns is provided via the keyboard shortcut Shift+Alt+S. Is that sufficient to resolve this issue?

**Comment by kapilvaishna** [comment]
@corivera yeah, that is fine to close the bug just make sure this shortcut key is there in your document.

We can change the column size with Shift +Alt + S key. so, closing the bug.

https://github.com/microsoft/azuredatastudio/assets/124254981/63251e28-84d5-436e-ab21-aa401ea73c6d
